components/step-section.js

```diff
@@ -0,0 +1,37 @@
+import { html, css } from "../html-css-utils.js";
+
+class StepSection extends HTMLElement {
+  constructor() {
+    super();
+    this.render();
+  }
+
+  render() {
+    this.attachShadow({ mode: "open" });
+    this.shadowRoot.innerHTML = this.createCss() + this.createHtml();
+  }
+
+  createCss() {
+    return css`
+      .step-section {
+        width: 80%;
+        max-width: 1550px;
+        margin: 0 auto;
+        padding: 100px 0;
+        @media only screen and (min-width: 1024px) {
+          padding: 149px 0;
+        }
+      }
+    `;
+  }
+
+  createHtml() {
+    return html`
+      <section class="step-section">
+        <slot></slot>
+      </section>
+    `;
+  }
```

Comment:
section 요소가 하나 밖에 없으니 그냥 타입 선택자를 사용해도 될 것 같습니다.
shadow dom의 장점을 최대한 살려서 불필요한 클래스의 사용을 최소화했으면 좋겠어요.

```suggestion
  createCss() {
    return css`
      section {
        width: 80%;
        max-width: 1550px;
        margin: 0 auto;
        padding: 100px 0;
        @media only screen and (min-width: 1024px) {
          padding: 149px 0;
        }
      }
    `;
  }

  createHtml() {
    return html`
      <section>
        <slot></slot>
      </section>
    `;
  }
```

Reply:
shadow dom의 장점을 살려서 불필요한 클래스 사용 방지하자는 점 적극 동의합니다! 다음 컴포넌트 만들 땐 요부분도 더 신경써보겠습니다. 의견 감사드립니다!